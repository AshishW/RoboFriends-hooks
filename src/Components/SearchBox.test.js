import SearchBox from "./SearchBox";
import React from "react";
import { shallow } from "enzyme";


it('renders searchbox component', ()=>{
    const SearchChange = jest.fn();
    expect(shallow(<SearchBox SearchChange = {SearchChange}/>)).toMatchSnapshot()
})

