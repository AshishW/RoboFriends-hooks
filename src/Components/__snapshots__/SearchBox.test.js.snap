// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders searchbox component 1`] = `ShallowWrapper {}`;
